homeassistant/components/ferienapidotde/binary_sensor.py

```diff
@@ -0,0 +1,135 @@
+"""Binary sensor to indicate if today is a german vacation day or not."""
+
+from datetime import timedelta
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.binary_sensor import BinarySensorDevice
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import CONF_NAME
+from homeassistant.exceptions import PlatformNotReady
+import homeassistant.helpers.config_validation as cv
+from homeassistant.util import Throttle
+
+REQUIREMENTS = ['ferien-api==0.3.1']
+
+_LOGGER = logging.getLogger(__name__)
+
+ALL_STATE_CODES = [
+    "BW", "BY", "BE", "BB", "HB", "HH", "HE", "MV", "NI", "NW", "RP", "SL",
+    "SN", "ST", "SH", "TH"
+]
+
+ATTR_START = 'start'
+ATTR_END = 'end'
+ATTR_NEXT_START = 'next_start'
+ATTR_NEXT_END = 'next_end'
+ATTR_VACATION_NAME = 'vacation_name'
+
+CONF_STATE_CODE = 'state_code'
+
+DEFAULT_NAME = 'Vacation Sensor'
+
+# Don't rush the api. Every 12h should suffice.
+MIN_TIME_BETWEEN_UPDATES = timedelta(hours=12)
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_STATE_CODE): vol.In(ALL_STATE_CODES),
+    vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string
+})
+
+SCAN_INTERVAL = timedelta(minutes=1)
```

Comment:
Why do we need to scan every minute if we're only updating the api every 12 hours? We can remove the throttle and instead set the scan interval to 12 hours.

Reply:
We need to re-evaluate the attributes which might update more often

Reply:
Ok.